comms/dht/src/config.rs

```diff
@@ -42,6 +42,10 @@ pub struct DhtConfig {
     /// Send to this many peers when using the propagate strategy
     /// Default: 4
     pub propagation_factor: usize,
+    /// The amount of seconds added to the current time (Utc) which will then be used to check if the message has
+    /// expired or not when processing the message
+    /// Default: 10800
+    pub saf_msg_validity: chrono::Duration,
```

Comment:
Don't use chrono::Duration for public interfaces. Rather use std::time::Duration and convert it further down. Without looking deeply into it or following the flow, we should be able to avoid using chrono (directly) at all for this feature, `std::time::Duration` for the validity setting and `Epoch::now()` to convert to `protobuf Timestamp`

Reply:
Think this is sorted now. Sorry had some confusion between two of your previous comments.